### PR TITLE
Form in template

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
Changed PR template for contributors to contain the new form

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
 <!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->
 
-## XSOAR [Partner](https://xsoar.pan.dev/docs/partners/become-a-tech-partner)?
-In case you are a technological partner:
-- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**.
+## Contributing to XSOARs content?
+Make sure to register your contribution - [contribution registration](https://forms.gle/XDfxU4E61ZwEESSMA)
```
Comment (Itay4) [inline .github/pull_request_template.md:4]:
```suggestion
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,10 +1,9 @@
 <!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->
 
-## XSOAR [Partner](https://xsoar.pan.dev/docs/partners/become-a-tech-partner)?
-In case you are a technological partner:
-- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**.
+## Contributing to XSOARs content?
+Make sure to register your contribution - [contribution registration](https://forms.gle/XDfxU4E61ZwEESSMA)  
 
-If you are not a technological partner (i.e. PANW employee, customer or individual developer), please delete this section.
+**The Pull Request review will only start after this form was filled**
```
Comment (Itay4) [inline .github/pull_request_template.md:6]:
```suggestion
**The Pull Request will be reviewed only after the contribution registration form is filled.**
```

**Comment by ShahafBenYakir** [comment]
@Itay4 @fvigo @anara123 Please have a look when you have a minute

**Review by Itay4 — APPROVED** [review]
looks good

**Review by anara123 — APPROVED** [review]

**Review by fvigo — APPROVED** [review]
ship it! 👍

**Pull request merged**